challenge-21/submissions/Sylinsic/solution-template.go

```
@@ -0,0 +1,143 @@
+        halfway := diff / 2
+        if arr[lower + halfway] == target {
+            return lower + halfway
+        }
+        
+	    switch {
+	        case arr[lower + halfway] > target:
+	            upper = lower + halfway
+            case arr[lower + halfway] < target:
+                lower = lower + halfway
+            default:
+                break
```

golangci-lint (2.12.2)

[error] SA4011: ineffective break statement. Did you mean to break out of the outer loop?

(staticcheck)